lib/addons/vpc-cni/index.ts

```diff
@@ -436,7 +443,8 @@ function populateVpcCniConfigurationValues(props?: VpcCniAddOnProps): Values {
       WARM_IP_TARGET: props?.warmIpTarget,
       WARM_PREFIX_TARGET: props?.warmPrefixTarget,
     },
-    enableNetworkPolicy: JSON.stringify(props?.enableNetworkPolicy)
+    enableNetworkPolicy: JSON.stringify(props?.enableNetworkPolicy),
+    enableWindowsIpam:JSON.stringify(props?.enableWindowsIpam)
```

Comment:
Space is missing after `:`

Reply:
@elamaran11  Updated the docs and added space.